applications/vanilla/settings/class.hooks.php

```diff
@@ -367,6 +367,15 @@ public function base_render_before($sender) {
         if ($sender->Menu) {
             $sender->Menu->addLink('Discussions', t('Discussions'), '/discussions', false, ['Standard' => true]);
         }
+        
+        if (!inSection('Dashboard')) {
+            // Spoilers assets
+            $sender->addJsFile('spoilers.js', 'dashboard');
+            $sender->addCssFile('spoilers.css', 'dashboard');
+            $sender->addDefinition('Spoiler', t('Spoiler'));
+            $sender->addDefinition('show', t('show'));
+            $sender->addDefinition('hide', t('hide'));
+        }
```

Comment:
Yeah, I had some hesitation there, too. I felt strange about adding it to the dashboard hooks and then saying 'Only add this if we're *not* in the dashboard'. On the other hand, the assets were located in the dashboard. I decided to move the assets to the Vanilla application instead, because that's where they're being used.